src/frontend/src/flows/confirmRegister.ts

```diff
@@ -67,20 +67,30 @@ const tryRegister = (
       confirmParagraph.innerHTML =
         "The value you entered is incorrect. A new challenge is generated.";
       requestCaptcha();
-    } else {
-      // Currently if something goes wrong we only tell the user that
+    } else if (result.kind == "registerNoSpace") {
+      // Currently, if something goes wrong we only tell the user that
       // something went wrong and then reload the page.
+      displayError({
+        title: "No Space Left",
+        message:
+          "We could not create an identity anchor because Internet Identity is at maximum capacity. Click 'ok' to reload.",
+        primaryButton: "Ok",
+      }).then(() => {
+        window.location.reload();
+      });
+    } else {
       displayError({
         title: "Something went wrong",
         message:
           "We could not create an identity anchor. You will find the full error message below. Click 'ok' to reload.",
         primaryButton: "Ok",
-        detail: JSON.stringify(result),
-      })
-        .then(() => {
-          window.location.reload();
-        })
-        .then(() => requestCaptcha());
```

Comment:
Looks like we're not requesting a new captcha in case of failure, where did that go?

Reply:
I removed it because the `window.location.reload()` makes it obsolete. On reload, we are on the welcome page again -> when we chose try again the captcha will be requested then.

Reply:
Mh, I really wonder what that was doing there. Bad copy paste I guess?